src/blocks/CTA/CTA.scss

```diff
@@ -19,58 +19,43 @@ $block: '.#{$namespace}cta';
 
     &__button {
```

Comment:
I saw a layout, why the whole item's name is button? It's not a button
<img width="384" alt="image" src="https://github.com/gravity-ui/blog-constructor/assets/22456330/3c14d889-8c55-46d5-866a-492762df8f93">

Reply:
agree